docs/openstack/backend_services_deployment.md

```diff
@@ -62,6 +62,25 @@ podified OpenStack control plane services.
 
 ## Pre-checks
 
+Get the database_connection and transport_url cells mappings on the source OSP environment, before those get renamed during
+the [databases adoption](mariadb_copy.md). Note those below to compare it later with the renamed post-adoption values.
+
+Firstly, check the templated view stored in the galera database:
+```bash
+podman exec -it galera-bundle-podman-0 mysql -uroot nova_api -e"select uuid,name,transport_url,database_connection,disabled from cell_mappings"  | grep '|'
+| uuid                                 | name    | transport_url                                                              | database_connection                                                                                                                    | disabled |
+| 00000000-0000-0000-0000-000000000000 | cell0   | none:///                                                                   | mysql+pymysql://{username}:{password}@192.168.122.99/nova_cell0?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo |        0 |
+| 58036269-e862-4b3d-943d-2dbd0fc2d915 | default | rabbit://{username}:{password}@standalone.ctlplane.localdomain:5672/?ssl=0 | mysql+pymysql://{username}:{password}@192.168.122.99/nova?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo       |        0 |
+```
+
+Then note its Nova representation (showing names and passwords):
+```bash
+podman exec -it nova_api nova-manage cell_v2 list_cells --verbose | grep '|'
+|   Name  |                 UUID                 |                                    Transport URL                                     |                                                               Database Connection                                                               | Disabled |
+|  cell0  | 00000000-0000-0000-0000-000000000000 |                                       none:///                                       | mysql+pymysql://nova:4w6xaYHjGSGJDv66z7xhHQioz@192.168.122.99/nova_cell0?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo |  False   |
+| default | 58036269-e862-4b3d-943d-2dbd0fc2d915 | rabbit://guest:FxEbJ5yKxaGiYthgI0PJFk3y1@standalone.ctlplane.localdomain:5672/?ssl=0 |    mysql+pymysql://nova:4w6xaYHjGSGJDv66z7xhHQioz@192.168.122.99/nova?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo    |  False   |
```

Comment:
@SeanMooney could you please confirm the new name for nova DB? post adtopion

Reply:
```
❯ oc rsh openstack-galera-0 mysql -u root --password=12345678 -e 'select * from nova_api.cell_mappings;'
Defaulted container "galera" out of: galera, mysql-bootstrap (init)
+---------------------+---------------------+----+--------------------------------------+-------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------+----------+
| created_at          | updated_at          | id | uuid                                 | name  | transport_url                                                                                                | database_connection                                            | disabled |
+---------------------+---------------------+----+--------------------------------------+-------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------+----------+
| 2023-10-10 11:31:22 | 2023-10-10 11:31:23 |  1 | 00000000-0000-0000-0000-000000000000 | cell0 | rabbit://                                                                                                    | mysql+pymysql://nova_cell0:12345678@openstack/nova_cell0       |        0 |
| 2023-10-10 11:31:43 | NULL                |  2 | 10e461ba-b9de-4545-b180-11e03915ca3f | cell1 | rabbit://default_user_WKaxHjAoXETXcRzQQX2:RVwgQpE3BvK6jnmOQ_uTVHFE2mnMJ5Sd@rabbitmq-cell1.openstack.svc:5672 | mysql+pymysql://nova_cell1:12345678@openstack-cell1/nova_cell1 |        0 |
+---------------------+---------------------+----+--------------------------------------+-------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------+----------+
```

Reply:
so what 
* "nova" -> "nova_cell1"
* "nova_cell0" -> "nova_cell0"
* "nova_api" -> "nova_api"

Also: 
```
❯ oc rsh openstack-galera-0 mysql -u root --password=12345678 -e 'show databases;'
Defaulted container "galera" out of: galera, mysql-bootstrap (init)
+--------------------+
| Database           |
+--------------------+
| cinder             |
| glance             |
| information_schema |
| keystone           |
| mysql              |
| neutron            |
| nova_api           |
| nova_cell0         |
| performance_schema |
| placement          |
+--------------------+
❯ oc rsh openstack-cell1-galera-0 mysql -u root --password=12345678 -e 'show databases;'
Defaulted container "galera" out of: galera, mysql-bootstrap (init)
+--------------------+
| Database           |
+--------------------+
| information_schema |
| mysql              |
| nova_cell1         |
| performance_schema |
+--------------------+
````